events_mgmt/src/test/java/com/adobe/event/management/ProviderServiceTest.java

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright 2017 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package com.adobe.event.management;
+
+import com.adobe.event.management.model.EventMetadata;
+import com.adobe.event.management.model.Provider;
+import com.adobe.event.management.model.ProviderInputModel;
+import com.adobe.ims.JWTAuthInterceptor;
+import com.adobe.ims.util.TestUtil;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import feign.RequestInterceptor;
+import org.junit.Rule;
+import org.junit.Test;
+import java.util.*;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertNotNull;
+
+public class ProviderServiceTest {
+
+  private static final String PROVIDER_ID = "619363ad-2461-4c1c-bb09-848f8b4d0a30";
+  private static final String SOURCE_ID = "urn:uuid:619363ad-2461-4c1c-bb09-848f8b4d0a30";
+
+  private static final String PROVIDER_MODEL_LABEL = "aio-lib-java Wiremock - test drive provider label";
+  private static final String PROVIDER_MODEL_DESCRIPTION = "aio-lib-java Wiremock - test drive provider description";
+  private static final String PROVIDER_MODEL_DOCS_URL = "https://localhost.com/test";
+
+  private static final String METADATA_EVENTCODE = "wiremock_eventcode";
+  private static final String METADATA_DESCRIPTION = "wiremock test description";
+  private static final String METADATA_LABEL = "test label";
+
+  @Rule
+  public WireMockRule wireMockRule = new WireMockRule(9999);
```

Comment:
the wiremock port could also be a constant as it's found in the response

Reply:
port constant has been added